src/components/MenuComponent.jsx

```diff
@@ -0,0 +1,318 @@
+            </div>
+          </div>
+
+          <div className='table__column'>
+            <div className='column__items'>
+              <div>0%</div>
+              <div>25%</div>
+              <div>50%</div>
+              <div>75%</div>
+              <div>100%</div>
+              <div>125%</div>
+              <div>150%</div>
+              <div>175%</div>
+              <div>200%</div>
+
+              <div className='bar1'>200%</div>
+              <div className='bar2'>25%</div>
+              <div className='bar3'>100%</div>
+            </div>
+          </div>
+        </Table>
+      </KarakterTable>
+    </KarakterWrapper>
+  );
+}
+
+function Knowledgecomponent() {
+  return <h2>KnowledgeComponent</h2>;
+}
+function Achievementscomponent() {
+  return <h2>AchievementsComponent</h2>;
+}
+function Questscomponent() {
+  return <h2>QuestsComponent</h2>;
+}
+function Historycomponent() {
+  return <h2>HistoryComponent</h2>;
+}
+
+const Menu = styled.nav`
+  height: 90px;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+
+  ul {
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+    background: #f97970;
+    padding: 15px 0;
+    place-items: center center;
+  }
+
+  li {
+    list-style-type: none;
+  }
+
+  li:hover::after {
+    position: absolute;
+    top: -125px;
+    left: 7%;
+    padding: 10px;
+    border-radius: 50px;
+    font: oblique small-caps bold 30px/21px Georgia, serif;
+    background: #fff;
+  }
+  li:nth-child(2):hover::after {
+    content: 'Karakter';
+    top: -75px;
+    left: 5%;
+  }
+  li:nth-child(3):hover::after {
+    content: 'Knowledge Book';
+    top: -75px;
+    left: 25%;
+  }
+  li:nth-child(4):hover::after {
+    content: 'Achievements';
+    top: -75px;
+    left: 45%;
+  }
+  li:nth-child(5):hover::after {
+    content: 'Quests';
+    top: -75px;
+    left: 65%;
+  }
+  li:nth-child(6):hover::after {
+    content: 'History';
+    top: -75px;
+    left: 85%;
+
+    @media only screen and (max-width: 900px) {
+      left: 81%;
+    }
+    @media only screen and (max-width: 600px) {
+      left: 73%;
+    }
+  }
+
+  .menu__lastitem {
+    grid-column-end: -1;
+  }
+  li a > i {
+    font-size: 45px;
+    color: #fbefd7;
+    width: 100%;
+  }
+  li a > i:hover {
+    color: #02132c;
+  }
+
+  li a:focus > i {
+    color: #02132c;
+  }
+`;
+
+// kakrkater
+
+const KarakterWrapper = styled.div`
+  background: #fff;
+  border-radius: 50px;
+  padding: 30px;
+  margin: 30px 0 0 0;
+  max-width: 1000px;
+  width: 50%;
+  margin: 0 10%;
+  @media only screen and (max-width: 900px) {
+    width: 70%;
+  }
+`;
+
+const KarakterContent = styled.div`
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-gap: 20px;
+`;
+
+const ImgWrapper = styled.div`
+  img {
+    width: 100%;
+  }
+  @media only screen and (max-width: 1100px) {
+    visibility: hidden;
+  }
+`;
+
+const ContentText = styled.div`
+  min-width: 350px;
+  margin: 10% 0 0 10%;
+
+  p > span {
```

Comment:
waarom verschillende spans in een p en dan toch display:block geven? kun je dan niet beter 3 p's gebruiken? of een andere structuur? Misschien is `dl` wel semantischer hier?

Reply:
leuk intressant was nieuw voor mij